dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/repository/Limit.java

```diff
@@ -15,38 +15,27 @@
 /**
  * Method signatures are copied from the jakarta.data.repository.Limit from the Jakarta Data repo.
  */
-public class Limit {
-    private final int max;
-    private final long start;
-
-    private Limit(long startAt, int maxResults) {
-        start = startAt;
-        max = maxResults;
-        if (start < 1)
-            throw new IllegalArgumentException("startAt: " + start);
-        if (max < 1)
-            throw new IllegalArgumentException("maxResults: " + max);
-    }
-
-    public int maxResults() {
-        return max;
+public record Limit(int maxResults,
```

Comment:
Just pointing out that the order of the parameters was switched, but I assume that was intentional.

Reply:
Yes, the order needs to match what is in the Jakarta Data beta 3 driver.